jsapp/js/components/formLanding.es6

```diff
@@ -264,9 +264,15 @@ export class FormLanding extends React.Component {
               </ui.PopoverMenu>
             </bem.FormView__cell>
             <bem.FormView__cell>
-              {chosenMethod != 'iframe_url' && chosenMethod != 'android' && this.state.deployment__links[chosenMethod] &&
-                <CopyToClipboard text={this.state.deployment__links[chosenMethod]} onCopy={() => notify('copied to clipboard')}>
-                  <button className='copy mdl-button mdl-button--colored'>{t('Copy')}</button>
+              {chosenMethod != 'iframe_url' && chosenMethod != 'android' &&
+               this.state.deployment__links[chosenMethod] &&
+                <CopyToClipboard text={this.state.deployment__links[chosenMethod]}
+                  onCopy={() => notify('copied to clipboard')}
```

Comment:
I know you don't want to change stuff unnecessarily, but maybe we can make it `t('copied to clipboard')`? Same below? 🙇‍♀